gcore/src/msg.rs

```diff
@@ -151,6 +152,43 @@ pub fn reply(payload: &[u8], gas_limit: u64, value: u128) -> MessageId {
     }
 }
 
+/// Finalizes current reply message.
```

Comment:
We use imperative tense despite it is not too Rust-doc-like.
```suggestion
/// Finalize current reply message.
```